airgun/entities/subscription.py

```diff
@@ -145,6 +145,48 @@ def delete(self, entity_name):
         self._wait_for_process_to_finish(
                 'Delete Upstream Subscription', has_manifest=True)
 
+    def read_header(self):
```

Comment:
doesn't Table object contain information about its columns headers?

Reply:
I can only find table.headers within view = self.navigate_to(self, 'All').  If there's a better way to display the header info than this, please advise on how that is implemented.

Reply:
Right
so why do you need separate method to read them?

Reply:
Here's my initial reasoning for needing a separate method:
1. I'm separating reading the headers and checking/unchecking the checkboxes for assertion purposes in ui_airgun/test_subscription.py
2. Expanding the test cases in the future may just need checking/unchecking the checkboxes without need of reading the headers.

After thinking about it, I see what you mean by not separating read header into a method because reason 1 and 2 wouldn't be a large blocker to include or not include the read header in there.

I'll play around with this a bit more and fix the changes based on your suggestion.